### DISCUSSION TITLE
Quad-issue processor model in Ripes

### DISCUSSION THREAD

**Discussion by Panagiotis-Chrysanthou** [mortbopet/Ripes#246] [General]
Hello, I am trying to dublicate the rv6s_dual.lib in Qt in order to make another model which will be quad issue. I already went into the processors CmakeList and created rv6s_quad.lib [create_vsrtl_processor(RISC-V rv6s_quad)]. How can I proceed into bringing/duplicating the files from the dual model into the quad model in order to start editing them? Also, any tips towards my goal would be very helpful! Thank you very much.

**Comment by mortbopet** [comment]
I would recommend you to take one of the processor models, duplicate the files (folder) for that model, rename it to your new model name, and then get started on figuring out your design! Don't focus too much on how the model looks (the visual layout) - you're probably going to be iterating a lot to fix any bugs in your design and each change to the top-level file in the model will need a layout change.

**Reply by Panagiotis-Chrysanthou** [reply]
Thank you a lot!
